asf_core_data/getters/mcs_getters/get_mcs_installations.py

```diff
@@ -133,3 +152,64 @@ def get_processed_installations_data(
             processed_installations_file_name,
         ),
     )
+
+
+def find_most_recent_mcs_installations_batch(epc_version: str = "none"):
+    """
+    Finds most recent batch of MCS installations data, joined with EPC data in specified format if specified.
+    Args:
+        epc_version: One of "none", "full", or "most_relevant".
+            - "none" returns just installation data
+            - "full" returns installation data with each property's entire EPC history attached
+            - "most_relevant" selects the most recent EPC from before the HP installation if one exists or the earliest
+            EPC from after the HP installation otherwise. Defaults to "none".
+    Returns:
+        str: Filename of most recent MCS installations (+ EPC) data.
+    """
+    bucket = s3.Bucket(bucket_name)
+    folder = mcs_processed_dir
+    file_list = [object.key for object in bucket.objects.filter(Prefix=folder)]
+    file_prefix = keyword_to_path_dict[epc_version].split("{")[0]
+    matches = [
+        filename
+        for filename in file_list
+        if ("/" + re.split("[0-9]", filename)[0] == file_prefix)
+    ]
+    try:
+        latest_version = max(matches)
+        print(f"Latest version available on S3: <{latest_version}>")
+        return latest_version
+    except ValueError:
+        print(f"No files found in {bucket} bucket for epc_version='{epc_version}'")
+
+
+def get_processed_installations_data_by_batch(
+    batch_date: str = "newest", epc_version: str = "none"
+) -> pd.DataFrame:
+    """
+    Get a specified version of the processed MCS installation (+ EPC) data (both domestic and non-domestic)
+    from S3 (either just MCS or MCS merged with EPC) using batch date.
+
+    Args:
+        batch_date: Date of desired batch of processed MCS installation data in YYMMDD format. Defaults to "newest"
+        which searches S3 for latest batch.
+        epc_version: One of "none", "full", or "most_relevant".
+            - "none" returns just installation data
+            - "full" returns installation data with each property's entire EPC history attached
+            - "most_relevant" selects the most recent EPC from before the HP installation if one exists or the earliest
+            EPC from after the HP installation otherwise. Defaults to "none".
+    Returns:
+        DataFrame: Processed MCS installations data (either merged or not merged with EPC)
+    """
+    if batch_date == "newest":
+        processed_installations_file_path = find_most_recent_mcs_installations_batch(
+            epc_version=epc_version
+        )
+    else:
+        file_prefix = keyword_to_path_dict[epc_version]
+        processed_installations_file_path = file_prefix.format(batch_date)[1:]
```

Comment:
just makes it easier to understand the [1:]
```suggestion
       # note that [1:] removes the first "/" in the file path
        processed_installations_file_path = file_prefix.format(batch_date)[1:] 
```